gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystem.java

```diff
@@ -218,18 +221,40 @@ public class GoogleHadoopFileSystem extends FileSystem implements IOStatisticsSo
 
   private ITraceFactory traceFactory = TraceFactory.get(/* isEnabled */ false);
 
+  /** Instrumentation to track Statistics */
+  ITraceFactory getTraceFactory() {
+    return this.traceFactory;
+  }
+
   /**
    * Constructs an instance of GoogleHadoopFileSystem; the internal GoogleCloudStorageFileSystem
    * will be set up with config settings when initialize() is called.
    */
-  public GoogleHadoopFileSystem() {}
+  public GoogleHadoopFileSystem() {
```

Comment:
We need this because we need to define storagestatistics at the time of creation of GHFS object. This way reduces code duplication.